requirements-dev.txt

```diff
@@ -0,0 +1,7 @@
+# Requirements you might want to install if you're working on this code.
+# The following are used by Makefile targets.
+discover==0.4.0
+pep8==1.5.7
+pyflakes==0.8.1
+tox==2.2.1
```

Comment:
Shouldn't `tox` be installed for everyone? A normal person might want to run tests too.